MANIFEST.in

```diff
@@ -1,7 +1,2 @@
-include dash_core_components/*.js
-include dash_core_components/*.js.map
-include dash_core_components/metadata.json
-include dash_core_components/package-info.json
 include README.md
 include LICENSE.txt
```

Comment:
We have a `LICENSE`, not `LICENSE.txt`... yet the rc1 tarball contains the file anyway, is that just included by default?

Reply:
huh, not sure what rule caused this to work anyway. But I removed the extension from MANIFEST.in in 6928a75